system_tests/caff_node_test.go

```diff
@@ -502,6 +508,107 @@ func TestEspressoCaffNodeUnfinalizedDelayedMessages(t *testing.T) {
 	Require(t, err)
 }
 
+// Test 1: Check if restart works on EspressoCaffNode without any TEE type specified, this will make sure that if fromBlock and hotshotBlock
```

Comment:
Awesome that we have this for the purposes of the tests. Does this mean that the block itself still needs to be signed during the tests?

Reply:
Yes! Its just that we use a private key instead of AWS KMS which I think is fine to test the functionality.